boards/nucleo-l433rc/Makefile.include

```diff
@@ -4,7 +4,7 @@ export CPU_MODEL = stm32l433rc
 
 # stdio is not available over st-link but on the Arduino TX/RX pins
 # A serial to USB converter plugged to the host is required
-PORT_LINUX ?= /dev/ttyUSB0
+PORT_LINUX ?= /dev/ttyACM0
```

Comment:
Maybe you can remove the comment above?

Reply:
Of course +1

Reply:
Even better: use the common configuration provided for nucleo64 boards ;)